lib/modules/apostrophe-schemas/index.js

```diff
@@ -1789,7 +1789,7 @@ module.exports = {
         string: function (req, data, name, object, field, callback) {
           var test = self.apos.launder.string(data[name]);
           if (tinycolor(test).isValid()) {
-            object[name] = test;
+            object[name] = data[name];
```

Comment:
`test` is laundered, `data[name]` is not; could have been coerced to a valid string from something-else that won't play nice. You still want to set `object[name]` to `test`